pom.xml

```
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.72</version>
+    <version>4.74</version>
```

Parent pom 4.75 is the most recent release now
```suggestion
    <version>4.75</version>
```